lib/common.js

```diff
@@ -1,7 +1,6 @@
 'use strict'
 
 const { common: debug } = require('./debug')
-const timers = require('timers')
 const url = require('url')
 const util = require('util')
 const http = require('http')
```

Comment:
Using the import from `timers` will cause `jest` to timeout, using the global ones will allow to mock the timers for `jest` as well